Adding note warning about ding not working (#28351)
home-assistant/home-assistant.github.io: source/_integrations/ring.markdown

ha_integration_type: integration
---

<p class='note'>

> This integration have a bug that makes it impossible to make automations based on visitors presing the doorbell. [Issue #88705](https://github.com/home-assistant/core/issues/88705)

Reviewer: ```suggestion
This integration currently has a bug that makes it impossible to create automations based on visitors pressing the doorbell. [Issue #88705](https://github.com/home-assistant/core/issues/88705)
```

> </p>

Reviewer 2: We don't document issues, please use and track the issue tracker for that.

Reason is that we have 2K open issue right now, if we would document them all (and have to remove them each time one is solved), it would become unmaintainable.

../Frenck
